presto-raptor-legacy/src/main/java/io/prestosql/plugin/raptor/legacy/RaptorTableHandle.java

```diff
@@ -38,8 +42,35 @@
     private final OptionalInt bucketCount;
     private final boolean organized;
     private final OptionalLong transactionId;
+    private final TupleDomain<RaptorColumnHandle> constraint;
+    private final Optional<List<String>> bucketAssignments;
     private final boolean delete;
 
+    public RaptorTableHandle(
```

Comment:
I think we can keep a single constructor. The `@JsonIgnore` should suffice to drop the assignments from serialization.